### PR TITLE
Migrate the documentation to staging

### LINKED ISSUE #1517: Convert pulp_container docs to new stack
The general workflow for this task can be found [here](https://hackmd.io/@pbrochad/SyfGeQc86#Content-Migration).

### PR DESCRIPTION
**Pull request by lubosmj**
closes #1517

### CODE REVIEW

**Comment by lubosmj** [comment]
image

File: staging_docs/admin/guides/change-allowed-artifacts.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Update Supported OCI Types
```
Comment (ipanova) [inline staging_docs/admin/guides/change-allowed-artifacts.md:1]: s/update/configure

File: staging_docs/admin/guides/migrate-permissions.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Migrate from Permissions
```
Comment (ipanova) [inline staging_docs/admin/guides/migrate-permissions.md:1]: # Migrate from Permissions to Roles

File: staging_docs/admin/guides/sign-image.md [hunk 3]
```diff
@@ -0,0 +1,368 @@
+# Image Signing
```
Comment (ipanova) [inline staging_docs/admin/guides/sign-image.md:1]: Sign Images.
since you started with imperative format maybe continue with it

**Comment by ipanova** [comment]
looks great!

File: staging_docs/user/tutorials/00-overview.md
Comment (pedro-psb) [inline staging_docs/user/tutorials/00-overview.md:1]: You MAY put this on `user/index.md` to have it be displayed in the top-level of the plugin namespace, which is generally where overviews are.
I've just added this behavior pulp-docs, so feel free to leave it for later.

File: staging_docs/admin/learn/tech-preview.md
Comment (pedro-psb) [inline staging_docs/admin/learn/tech-preview.md:1]: Would *may* put this in admin/reference, I've merged the changes that will correctly fetch them.

**Review by pedro-psb — CHANGES_REQUESTED** [review]
Look great!
Apart from the user/tutorial which I commented in the Matrix, the other suggestions are just so you know you can, feel free to ignore.

File: staging_docs/user/tutorials/00-overview.md
Comment (lubosmj) [reply staging_docs/user/tutorials/00-overview.md:1]: I tried moving it to `user/index.md`. But, I cannot access this site from anywhere.

File: staging_docs/admin/learn/tech-preview.md
Comment (lubosmj) [reply staging_docs/admin/learn/tech-preview.md:1]: I left it untouched. We already have such a structure in pulpcore. Will move it later.

File: staging_docs/user/tutorials/00-overview.md
Comment (pedro-psb) [reply staging_docs/user/tutorials/00-overview.md:1]: I'll have a look on that later.

**Review by pedro-psb — APPROVED** [review]

**Review by ipanova — APPROVED** [review]

**Pull request merged**